README.md

```diff
@@ -205,7 +167,7 @@ Suppose you want to delete an image called `my-image-name`. You can then delete
 docker image rm my-image-name
```

Comment:
Actually it does, because if someone for some reason want's to cleanup a single image they can do it by following these steps, i think it's ok to keep it

Reply:
but it's not really relevant to the lumino tests installation guide, is it? it's more of a docker thing imo.

@mauricioirace wdyt?

Reply:
Well yeah we could remove it

Reply:
Done